Update installation.mdx (#6653)
RasaHQ/rasa: docs/docs/installation.mdx

cd rasa
poetry install
```

> ## Upgrading Rasa Packages

Reviewer: ```suggestion
## Upgrading Versions
```

> Upgrade an already installed Rasa packages to the latest version from PyPI:

Reviewer: ```suggestion
To upgrade your installed version of Rasa Open Source to the latest version from PyPI:
```

```bash
pip install --upgrade rasa
```

> To Download a specific version, use required PyPI package version number:

Reviewer: ```suggestion

To download a specific version, specify the version number:
```
